benchmark/C/Savina/PingPongDistributed.lf

```diff
@@ -12,7 +12,7 @@
  * 
  * Unthreaded: 97 msec
  * Threaded: 265 msec
- * Distributed: 53 seconds
+ * Distributed: 6 seconds
```

Comment:
While it's a good idea to write these numbers down, this is probably not the best place to put them. We should probably create a shared spreadsheet (or even a spot on the wiki) where we can more carefully document which optimizations were involved, which platform it was executed on, what hardware, etc.

Reply:
I have started a Benchmarks wiki page and put pointers to it in the benchmarks: 
https://github.com/icyphy/lingua-franca/wiki/Benchmarks

Reply:
We do have a [NVIDIA Jetson AGX Xavier](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/) if you think benchmarks on a board like that could also be interesting. The C target at least works fine.

Reply:
Ah, yes, that would be quite interesting.  My guess is we would have to tune the code to get good performance...